gravitee-apim-console-webui/src/management/settings/top-apis/migrated/top-apis.component.html

```diff
@@ -0,0 +1,122 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<mat-card>
+  <mat-card-content>
+    <div class="header">
+      <div class="header__title">
+        <h3>Top APIs</h3>
+        <p>Showcase your APIs on the Developer Portal homepage</p>
+      </div>
+
+      <div class="header__actions">
+        <button
+          class="add-button"
+          (click)="addTopApi()"
+          mat-raised-button
+          color="primary"
+          *gioPermission="{ anyOf: ['environment-settings-c'] }"
+        >
+          <mat-icon>add</mat-icon>
+          Add Top API
+        </button>
+      </div>
+    </div>
+
+    <table mat-table [dataSource]="topApisList">
```

Comment:
What do you think about use the gio-table-wrapper  in order to have the same table and also pagination + search (even if only on the front side for the moment) ?

Reply:
I like idea of having the same look, however table in this case does not need pagination / search. Its simple , and do not contain many elements and will not. Implementing all this just to have similarity with other pages is bad idea in my opinion.

Reply:
solution for tables may be adding mode in which we remove this whole section, and live simple table :
![image](https://github.com/gravitee-io/gravitee-api-management/assets/135582189/6fb19372-b897-476b-9c80-2e27df80c5d9)